### PR TITLE
Feature/fgs 213

### PR DESCRIPTION
**Pull request by MatthewWaanders**

### CODE REVIEW

File: drivers/FGS-213/device.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const Homey = require('homey');
+const ZwaveDevice = require('homey-meshdriver').ZwaveDevice;
+
+class FibaroSingleSwitchTwoDevice extends ZwaveDevice {
+	
+	onMeshInit() {
+		this._S1Trigger = new Homey.FlowCardTriggerDevice('FGS-213_S1').registerRunListener(this._switchTriggerRunListener).register();
```
Comment (RobinBol) [inline drivers/FGS-213/device.js]: Indentation

File: drivers/FGS-213/device.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        this._S2Trigger = new Homey.FlowCardTriggerDevice('FGS-213_S2').registerRunListener(this._switchTriggerRunListener).register();
+        this._resetMeter = new Homey.FlowCardAction('FGS-213_reset_meter').registerRunListener(this._resetMeterRunListener).register();
+
+        this.registerCapability('onoff', 'SWITCH_BINARY');
+        this.registerCapability('measure_power', 'METER');
+        this.registerCapability('meter_power', 'METER');
+
+        this.registerSetting("53", (value) => {
```
Comment (RobinBol) [inline drivers/FGS-213/device.js]: Settings with numbers as id's are not validated successfully anymore with the new athom-cli or does this validate?

File: drivers/FGS-213/driver.compose.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+  "id": "FGS-213",
+  "name": {
+    "en": "Single Switch 3",
```
Comment (RobinBol) [inline drivers/FGS-213/driver.compose.json]: 3?:p

File: drivers/FGS-213/driver.settings.compose.json [hunk 4]
```diff
@@ -0,0 +1,157 @@
+[
+  {
+    "$extends": "saveState",
+    "$id": "9",
```
Comment (RobinBol) [inline drivers/FGS-213/driver.settings.compose.json]: See comment above, settings with number id's should they be changed (even though it breaks compatibility)?

File: drivers/FGS-213/driver.settings.compose.json [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        "id": "0",
+        "label": {
+          "en": "Momentary Switch",
+          "nl": "Korstondige schakelaar"
+        }
+      },
+      {
+        "id": "1",
+        "label": {
+          "en": "Toggle Switch (Closed = On, Open = Off)",
```
Comment (RobinBol) [inline drivers/FGS-213/driver.settings.compose.json]: Remove text between brackets also for option below.

File: drivers/FGS-213/driver.settings.compose.json [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+        "label": {
+          "en": "Toggle Switch (Changes State)",
+          "nl": "Tuimel Schakelaar (Veranderd Status)"
+        }
+      }
+    ],
+    "value": "2"
+  },
+  {
+    "type": "group",
```
Comment (RobinBol) [inline drivers/FGS-213/driver.settings.compose.json]: Is it really necessary to have two different power groups? seems a bit too much

File: drivers/FGS-213/driver.settings.compose.json [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+          "min": 0,
+          "max": 100
+        },
+        "value": 20
+      },
+      {
+        "id": "51",
+        "type": "number",
+        "label": {
+          "en": "Wattage report, minimum Time",
```
Comment (RobinBol) [inline drivers/FGS-213/driver.settings.compose.json]: ',' in labels is not really nice, also capitalisation

File: drivers/FGS-213/driver.settings.compose.json (hunk 6 above)
Comment (MatthewWaanders) [reply drivers/FGS-213/driver.settings.compose.json]: Fixed

File: drivers/FGBS-001/device.js [hunk 8]
```diff
@@ -73,8 +73,8 @@ class FibaroUniversalBinarySensor extends ZwaveDevice {
 		this.registerCapability('alarm_generic.contact2', 'SCENE_ACTIVATION', {
 			report: 'SCENE_ACTIVATION_SET',
 			reportParser: (report) => this._alarmGenericReportParser(report, 2),
-            reportParserOverride: true,
-        });
+			reportParserOverride: true,
```
Comment (RobinBol) [inline drivers/FGBS-001/device.js]: Ah, and they are also committed here:P (see other PRs) It is best to keep your PRs clean!

**Pull request merged**